ui/src/components/drafts/components/FileItem.js

```diff
@@ -48,7 +45,8 @@ class FileItem extends React.Component {
     super(props);
     this.state = {
       hover: false,
-      menu: false
+      menu: false,
+      publishedUrl: "/published/:id" === this.props.match.path
```

Comment:
Not correct. If URL is something like this `/published/:id/runs`, it won't work and display the button

Reply:
Should the check against a list with the available route paths which include the published,
```/published/:id```, ```/published/:id/runs/```, ```/published/:id/runs/create/``` enough?